gensim/models/_fasttext_bin.py

```diff
@@ -176,7 +194,8 @@ def _load_vocab(fin, new_format, encoding='utf-8'):
         raise NotImplementedError("Supervised fastText models are not supported")
     logger.info("loading %s words for fastText model from %s", vocab_size, fin.name)
 
-    _struct_unpack(fin, '@1q')  # number of tokens
+    ntokens, = _struct_unpack(fin, '@q')  # number of tokens
```

Comment:
Tricky one. `struct.unpack` always returns tuple, because it can produce multiple values from a given format. So in this case a call produces one-uple `(ntokens,)`, coma converts it to raw `ntokes`.

Changed to much more readable `struct.unpack(fin, '@q')[0]`